### PR TITLE
Add How-to reference on using logpdf built with another framework

### LINKED ISSUE #318: Add "How to use custom gradients?" documentation
It's very simple: https://jax.readthedocs.io/en/latest/notebooks/Custom_derivative_rules_for_Python_code.html#python-debugging

### PR DESCRIPTION
**Pull request by rlouf**
Use an Aesara model compiled to Numba with Blackjax. The example is currently very slow, even after calling `jax.jit` on the NUTS kernel, even though the Numba logp evaluates faster thant its JAX equivalent.

Closes #318

### CODE REVIEW

**Review by junpenglao — APPROVED** [review]

**Comment by rlouf** [comment]
We can do better by using custom XLA calls for the function, which will prevent back-and-forth with Python in the inner loop. I had an inkling this is what was happening, and this was confirmed by Sharad Vikram who also shared an example of such custom calls:

https://github.com/jax-ml/jax-triton/blob/main/jax_triton/triton_call.py

We´d need to add at least a note about this and maybe, better, an example.
